.github/workflows/translations-sync.yml

```diff
@@ -3,27 +3,27 @@
 
 # See translations-upload.yml for automation to upload our source content
 # See translations-pr-lint.yml for quality control we conduct on ingress of new translations.
-name: Crowdin Download
+name: Crowdin Translations Sync
 
 on:
-  workflow_dispatch: # Allow running when we want to, for events such as urgent translation mistakes or 100% completed languages
+  workflow_dispatch: # Manual trigger for urgent fixes
   schedule:
-    - cron: '0 5 * * 5' # At 05:00 on Fridays. This guarantees that we have the 72 hour weekend time to review translations.
+    - cron: '0 5 * * 5' # Fridays at 05:00 UTC, allowing weekend review time
 
 # Cancel any runs on the same branch
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-permissions:
-  contents: read
+permissions: {}
+
+env:
+  COMMIT_MSG: 'chore: sync translations from crowdin'
+  HEAD_REF: chore/crowdin
 
 jobs:
-  synchronize-with-crowdin:
+  sync-translations:
```

Comment:
I am not aware of a standard, so why change? 

see 
- https://github.com/nodejs/nodejs.org/blob/main/.github/workflows/translations-pr-lint.yml#L30
- https://github.com/nodejs/nodejs.org/blob/main/.github/workflows/pull-request-label.yml#L29
- https://github.com/nodejs/nodejs.org/blob/main/.github/workflows/pull-request-label.yml#L29

there are other kebab cases too